CORE/world_extensions/core_world.rb

```diff
@@ -15,6 +15,12 @@ def create_world
         @validation_engine = ValidationEngine.new(self)
         log_header
         set_data_target
+        set_default_watir_timeout
+    end
+
+    def set_default_watir_timeout
+      require 'Watir'
+      Watir.default_timeout = @configuration['DEFAULT_ELEMENT_TIMEOUT']
```

Comment:
Should this be a ternary that has a default in case the value is missing from the user config file?

Reply:
Seems to default to 30 if the value given is `nil`, so doesn't seem to be a problem anyway.

Reply:
Any reason we're requiring `Watir` again, when we have `require 'watir'` at the top of file?